lib/routes.js

```diff
@@ -24,7 +24,8 @@ const defaults = {
 		national:    true,
 		regionalExp: true,
 		regional:    true
-	}
+	},
+	bahncard: 		0 //25_1: 1, 25_2: 2, 50_1: 3, 50_2: 4
```

Comment:
I would prefer a name for every bahncard option. E.g:
hafas.NOBAHNCARD
hafas.BAHNCARD251
hafas.BAHNCARD252
hafas.BAHNCARD501
hafas.BAHNCARD502
hafas.VORTEILSCARD
hafas.HALBTAXABORAILPLUS
hafas.HALBTAXABO
hafas.VOORDEELURENABORAILPLUS
hafas.VOORDEELURENABO
hafas.SHCARD
hafas.CHGENERALABO